specification/subscription/resource-manager/Microsoft.Subscription/preview/2019-10-01-preview/subscriptions.json

```diff
@@ -868,6 +964,71 @@
           "readOnly": true
         }
       }
+    },
+    "TenantPolicy": {
+      "description": "Tenant policy.",
+      "properties": {
+        "blockSubscriptionsLeavingTenant": {
```

Comment:
thoughts on just having a single enum property? E.g.:
```
"subscriptionTransfers": "NotAllowed | Allowed | IncomingOnly | OutgoingOnly"
```

Reply:
If you go down this path, make sure each value of the enum has a description that explains what it means (specified via the x-ms-enum extension)

---
In reply to: [554128860](https://github.com/Azure/azure-rest-api-specs/pull/11672#discussion_r554128860) [](ancestors = 554128860)

Reply:
Our backend has boolean logic setup for this, which would need additional conversions